<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1023</width>
    <height>650</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>1000</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="testButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate QR Code</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>551</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>http://www.google.com/</string>
    </property>
   </widget>
   <widget class="QLabel" name="qrLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>541</width>
      <height>421</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>800</width>
      <height>800</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1023</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
